bookmarket-backend/src/main/java/edu/oakland/sophomoreproject/dependencies/sqlite/listings/ListingsTableAccessor.java

```diff
@@ -72,6 +69,22 @@ public Listing getListingById(int listingId) {
 	/// this can be done by doing `INSERT INTO listings ... RETURNING listing_id`
 	/// and parsing the `listing_id` column it returns with `results.getString("listing_id")`
 	public void createListing(ListingWithoutId listingWithoutId) {
-		// TODO
+		String sql ="INSERT INTO listings (title, author, description, classSubject, price, condition, createdAt, saleAvailability, sellerId" + "VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?)";
+
+		Connection connection = getDatabaseConnection();
+		PreparedStatement sqlQuery = connection.prepareStatement(sql);
+
+		sqlQuery.setString(1, title);
```

Comment:
Where are these variables `title`, `author`, `classSubject`, etc. coming from? Right now from nowhere (you should see a red squiggly error in intellij). These fields should be taken from the `listingWithoutId` parameter in this function in line 71